src/extension/prompts/node/base/terminalAndTaskState.tsx

```diff
@@ -117,11 +117,12 @@ export class TerminalAndTaskStatePromptElement extends PromptElement<TerminalAnd
 					)}
 				</>
 			);
-
+			const renderedTasks = renderTasks();
```

Comment:
How would this fix the bug? The _data_ (eg `resultTasks`) has already been computed by this point, and the problem is with collecting the tasks and terminals, no?

Reply:
Idk - I was seeing a strange result before this change and not seeing it after

Reply:
This code is equivalent as far as I can tell. I'm sure there's an element of randomness here